pkg/storage/replica_rangefeed.go

```diff
@@ -431,6 +434,21 @@ func (r *Replica) handleClosedTimestampUpdateRaftMuLocked() {
 	if closedTS.IsEmpty() {
 		return
 	}
+
+	c := r.raftMu.rangefeed.Config
+	slowClosedTSThreshold := 5 * closedts.TargetDuration.Get(&r.ClusterSettings().SV)
+	if d := timeutil.Since(closedTS.GoTime()); d > slowClosedTSThreshold {
+		c.Metrics.RangeFeedClosedTimestampBehind.Inc(1)
+
+		// WIP there should probably be a replica-wide limiter on the log and the
+		// call to `EmitMLAI`.
+		log.Infof(ctx, "RangeFeed %s closed timestamp %s is behind by %s", c.Span, closedTS, d)
+		// Attempt to kick the closed timestamp in case it's stuck.
+		//
+		// WIP is it okay to call this while holding raftMu?
+		r.EmitMLAI()
```

Comment:
My hunch is that this should not be emitting an MLAI but instead should be issuing a Request like https://github.com/cockroachdb/cockroach/blob/5c751820f1765dd997d2bebf3eb0f6e8e31af8e8/pkg/storage/replica_follower_read.go#L60

Reply:
I don't think it's reasonable to emit an MLAI unless we know for sure that we're the lease holder